images/Containerfile

```diff
@@ -29,6 +30,7 @@ RUN dnf install \
         pciutils \
         procps-ng \
         python3 \
```

Comment:
Can we remove python3 then?

Reply:
no. `dnf` depends on `python3`. We cannot uninstall that (it's probably in the container to begin with). Even if it were possible, it seems a bad idea to drop the "system" python (that is, the version that is deemed to be suitable for all other packaged python3 programs of the distro).

Reply:
ok